ui/src/app/apps/apps.service.ts

```diff
@@ -102,6 +102,14 @@ export class AppsService {
 
   }
 
+  unregisterMultipleApps(appRegs: AppRegistration[]): Observable<Response[]> {
+    const observables: Observable<Response>[] = [];
+    for (const appReg of appRegs) {
+      observables.push(this.unregisterApp(appReg));
+    }
+    return Observable.forkJoin(observables);
```

Comment:
@jvalkeal Yes we need to polish those things as we go along and add catch-statements in scenarios where we need to have some "continue on error" logic.

E.g. see here: https://stackoverflow.com/a/38061516/835934

In this particular use-case, I think, we are borderline-ok.